src/metabase/api/automagic_dashboards.clj

```diff
@@ -20,4 +22,9 @@
   [id]
   [(magic/automagic-dashboard (Table id))])
 
+(api/defendpoint GET "/analize/metric/:id"
```

Comment:
@salsakran is this the `GET` endpoint in question?

Reply:
Yup. There will eventually be a set of endpoints for metrics, segments, tables, etc. I'd just worry about getting it working for this one roundtrip